pages/api/characters/[chainLabel]/[characterId]/update.ts

```diff
@@ -24,20 +24,10 @@ const verifyCharacterPlayer = async (
 
   if (error) {
     console.error('Error getting character', error);
-    return false;
+    return JSON.stringify(error);
```

Comment:
I'd say we just throw the error here as errors are already caught and sent back as 500s